scripts/template_upload_all.py

```diff
@@ -378,6 +382,8 @@ def main():
 
         print("TEMPLATE_UPLOAD_ALL:------End of {} upload on: {}--------".format(
             kwargs['template_name'], provider_type))
+    else:
```

Comment:
if we made it all the way through the for loop without returning, we never matched. Without this the 'failure' there is silent